pai-management/paiLibrary/clusterObjectModel/paiObjectModel.py

```diff
@@ -314,7 +315,7 @@ def serviceParse(self):
 
         for host in self.rawData["clusterConfiguration"]["machine-list"]:
             hostname = host["hostname"]
-            self.labelExpend(host)
+            #self.labelExpend(host)
             host["nodename"] = host["hostip"]
```

Comment:
If no use anymore, suggest to delete it to avoid confusion.

Reply:
Ok, removed.